activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Define `unfreeze_time` alias method of `travel_back` in `ActiveSupport::Testing::TimeHelpers`.
```

Comment:
The idea here was

> Define `unfreeze_time` as an alias of `travel_back` in `ActiveSupport::Testing::TimeHelpers`.

"alias method" doesn't sound well because "alias" is actually a noun.

Reply:
Right, it now reads `Define `unfreeze_time` alias of `travel_back` in `ActiveSupport::Testing::TimeHelpers`.`